sos/plugins/cs.py

```diff
@@ -1,128 +1,140 @@
-# Copyright (C) 2007-2010 Red Hat, Inc., Kent Lamb <klamb@redhat.com>
-#                                        Marc Sauton <msauton@redhat.com>
-#                                        Pierre Carrier <pcarrier@redhat.com>
```

Comment:
You can't just nuke everyone else's copyright. I get that there is substantive churn in this patch, but a lot of it is just e.g. jiggling quote styles. Please add your own declaration of copyright if you wish to but leave the previous authors intact (unless you've been in contact with them and they've asked for their details to be removed).